example/c/include/diplomat_runtime.h

```diff
@@ -4,6 +4,7 @@
 #include <stdint.h>
```

Comment:
Question: does CI test to make sure that all the header files compile?

Reply:
Nope! Only ones transitively reachable by the test includes. We ought to have a test that's just `#include` everything + run under g++, or stuff a bunch of includes in one of the tests. I didn't bother to try and fix this yesterday though.

For C2/CPP2 I intend to test self-containedness (we already do for C2) so that _is_ tested.